rds-update-rdsh-collection/azuredeploy.json

```diff
@@ -0,0 +1,345 @@
+﻿{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+
+        "rdshCollectionName": {
+            "type": "string",
+            "defaultValue": "Desktop Collection",
+            "metadata": {
+                "description": "Name of the RDSH collection to update"
+            }
+        },
+
+        "rdshTemplateImageUri": {
+            "type": "string",
+            "metadata": {
+                "description": "URI for the template VHD to use for RDSH instances. For example, https://rdsstorage.blob.core.windows.net/vhds/MyWindowServerOSImage.vhd"
+            }
+        },
+        "rdshUpdateIteration": {
+            "type": "string",
+            "metadata": {
+                "description": "A unique designator representing update iteration; newly created RDSH instances will have their names in the form: 'rdsh-&lt;index&gt;&lt;iteration&gt;'"
+            }
+        },
+        "rdshNumberOfInstances": {
+            "type": "int",
+            "defaultValue": 2,
+            "metadata": {
+                "description": "Number of RD Session Hosts"
+            }
+        },
+        "rdshVmSize": {
```

Comment:
this is the same standard we're applying to all templates, this one doesn't seem to be unique in any sense...  And I have yet to have feedback that this is too challenging for the audience (and have plenty of feedback on the other side).  Happy to chat w/ your PM if it helps.